src/styles/blocks/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 49px 40px;
+
+  border-top: 1px solid $border-color;
+
+  grid-template-columns: repeat(auto-fit, 200px);
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(auto-fit, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(auto-fit, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(auto-fit, 1fr);
```

Comment:
The media queries for `min-width: 488px`, `min-width: 768px`, and `min-width: 1024px` all set `grid-template-columns` to `repeat(auto-fit, 1fr)`, which is redundant. Since they are all setting the same value, you can remove the first two media queries and keep only the one for `min-width: 1024px` to reduce redundancy and keep the code clean.